packages/storybook/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@nrwl/storybook",
+  "version": "0.0.1",
+  "description": "Storybook plugin for Nx",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nrwl/nx.git"
+  },
+  "keywords": [
+    "Monorepo",
```

Comment:
I don't know if this one applies. Monorepos are supported with this but this package can be used in a normal Angular CLI workspace